### ISSUE TITLE
Trying to migrate old config

### ISSUE THREAD

**Issue by sudoshindo** [fastfetch-cli/fastfetch#699]
I am trying to migrate an old config that I copied from another PC.

The first issue I encountered was `Error: unknown option: --gpu-hide-integrated`.

I tried searching the repository for a sample config file that has all the options. I wanted to use it as a reference to do manual edits of outdated options. I struggled to find one, or I just don't know what the config file has been renamed to.

I commented-out `--gpu-hide-integrated` and it "kind of works" now. But now there is a bug in the output of Themes and Icons.

```
Theme: {5}
Icons: {3}
```

**This is the config I'm trying to use in a zip file in case you want to try identifying how old the config is:**
[oldconfig.zip](https://github.com/fastfetch-cli/fastfetch/files/14021534/oldconfig.zip)

**Comment by CarterLi** [comment]
> --gpu-hide-integrated

It was renamed to `--gpu-hide-type integrated`. Note this option only works with `--gpu-force-vulkan` in Linux.

> Theme: {5}
> Icons: {3}

Support of `--theme-format` and `--icons-format` were removed long ago [in 1.11.2](https://github.com/fastfetch-cli/fastfetch/releases/tag/1.11.2) when porting these modules to Windows, because `GTK` and `QT` stuff are Linux specific. You have to use older version if you want to customize them.

**Comment by sudoshindo** [comment]
Quick side note:

I am on version...
```
fastfetch --version
fastfetch 2.2.3 (x86_64)
```

When I told fastfetch to generate a new *conf* formatted config I found that this chunk hasn't been updated (around line 270):
```
# GPU hide options
# Sets whether to hide certain gpu types
# Must be either true or false
# Default is false.
#--gpu-hide-integrated
#--gpu-hide-discrete
```

**Comment by CarterLi** [comment]
Yeah. The entire function of conf formatted config generation was removed several versions ago.

**Comment by CarterLi** [comment]
https://github.com/fastfetch-cli/fastfetch/releases

**Comment by sudoshindo** [comment]
Wow, jumped several minor versions in a month.

**Comment by sudoshindo** [comment]
My config is in a working state now.
Does the latest version completely drop support for the `*.conf` format?
If it does I should probably use that migrate flag.
Feel free to close this after answering that last question.

**Comment by CarterLi** [comment]
You should migrate to jsonc. The .conf config format will be completely removed in 3.0

**Comment by sudoshindo** [comment]
Migration done, dotfiles updated, closing issue.

Thanks for helping.
